applications/nrf_desktop/README.rst

```diff
@@ -361,7 +361,7 @@ Depending on what development kit you use, you need to select the respective con
 
       .. table-from-rows:: /includes/sample_board_rows.txt
          :header: heading
-         :rows: nrf52840dk_nrf52840, nrf52833dk_nrf52833
+         :rows: nrf52840dk_nrf52840, nrf52833dk_nrf52833, nrf52833dk_nrf52820
```

Comment:
Shouldn't this come under 'HID dongle' instead of 'DK' section?

Reply:
No because we use dk board in this case which emulates dongle.